auth0/src/main/java/com/auth0/android/request/internal/RequestFactory.java

```diff
@@ -47,7 +47,7 @@
     private static final String AUTHORIZATION_HEADER = "Authorization";
     private static final String USER_AGENT_HEADER = "User-Agent";
     private static final String ACCEPT_LANGUAGE_HEADER = "Accept-Language";
-    private static final String CLIENT_INFO_HEADER = Telemetry.HEADER_NAME;
+    private static final String CLIENT_INFO_HEADER = Auth0UserAgent.HEADER_NAME;
```

Comment:
should we rename the constant as well? (don't just accept this as you'd have to update its usages as well)

```suggestion
    private static final String AUTH0_USER_AGENT_HEADER = Auth0UserAgent.HEADER_NAME;
```

Reply:
New commit pushed to rename as suggested.